src/test/java/org/broadinstitute/consent/http/db/LibraryCardDAOTest.java

```diff
@@ -125,6 +126,96 @@ void testFindLibraryCardByIdNegative() {
     assertNull(cardFromDAO);
   }
 
+  @Test
+  void testFindLibraryCardDaaById() {
+    LibraryCard card = createLibraryCard();
+    Integer dacId = dacDAO.createDac(RandomStringUtils.randomAlphabetic(5), RandomStringUtils.randomAlphabetic(5), "",  new Date());
+    Integer daaId = daaDAO.createDaa(card.getUserId(), new Date().toInstant(), card.getUserId(), new Date().toInstant(), dacId);
+    DataAccessAgreement daa = daaDAO.findById(daaId);
+    card.addDaa(daa.getDaaId());
+    card.addDaaObject(daa);
+    libraryCardDAO.createLibraryCardDaaRelation(card.getId(), daaId);
+    Integer id = card.getId();
+    LibraryCard cardFromDAO = libraryCardDAO.findLibraryCardDaaById(id);
+
+    assertEquals(cardFromDAO.getUserId(), card.getUserId());
+    assertEquals(cardFromDAO.getUserName(), card.getUserName());
+    assertEquals(cardFromDAO.getUserEmail(), card.getUserEmail());
+    assertEquals(cardFromDAO.getCreateUserId(), card.getCreateUserId());
+    assertEquals(cardFromDAO.getCreateDate(), card.getCreateDate());
+    assertEquals(cardFromDAO.getDaaIds(), card.getDaaIds());
+    DataAccessAgreement daaFromDAO = cardFromDAO.getDaas().get(0);
+    assertEquals(daaFromDAO.getDaaId(), daa.getDaaId());
+    assertEquals(daaFromDAO.getCreateUserId(), daa.getCreateUserId());
+    assertEquals(daaFromDAO.getCreateDate(), daa.getCreateDate());
+    assertEquals(daaFromDAO.getUpdateUserId(), daa.getUpdateUserId());
+    assertEquals(daaFromDAO.getUpdateDate(), daa.getUpdateDate());
+    assertEquals(daaFromDAO.getInitialDacId(), daa.getInitialDacId());
+  }
+
+  @Test
+  void testFindLibraryCardDaaByIdMultipleDaas() {
```

Comment:
This test does all the same work as `testFindLibraryCardDaaById` so we don't really need both. I suggest dropping the first test and keeping this one since we're testing more functionality here.